File: experiment_jp_test/mutants/for__0__PROBABILITY_REACHING_POINT_2_3_STEPS_TIME.java
static float find_prob ( int N , float P ) { double dp [ ] = new double [ N + 1 ] ; dp [ 0 ] = 1 ; dp [ 1 ] = 0 ; dp [ 2 ] = P ; dp [ 3 ] = 1 - P ; return ( ( float ) ( dp [ N ] ) ) ; }

File: experiment/java-python/test/j2pyrawpy/for__0__PROBABILITY_REACHING_POINT_2_3_STEPS_TIME.py
#!/usr/bin/env python
""" generated source for module for__0__PROBABILITY_REACHING_POINT_2_3_STEPS_TIME """
class X(object):
    """ generated source for class X """
    @classmethod
    def find_prob(cls, N, P):
        """ generated source for method find_prob """
        dp = [None]*N + 1
        dp[0] = 1
        dp[1] = 0
        dp[2] = P
        dp[3] = 1 - P
        return (((dp[N])))

